### DISCUSSION TITLE
Voice bubble

### DISCUSSION THREAD

**Discussion by AliShahesmaeili** [dotnet/maui#4332] [Q&A]
I'm building a voice assistant and I need it to float on the Android screen like this:

(Telegram Messenger)

https://user-images.githubusercontent.com/67000626/151212626-b871e052-4ca8-4208-9c8f-df77311c01a5.mp4

And like Voice Typing in Windows 10/11:
Inactive:
![image](https://user-images.githubusercontent.com/67000626/151213137-c169a0fa-6a4b-4708-8130-68bf9ee3a8b0.png)

Active:
![image](https://user-images.githubusercontent.com/67000626/151213277-7217df33-46a8-45c1-9d65-17667e45f828.png)

How to create these floating windows/bubble in MAUI?

**Comment by Rickard87** [comment]
Did you find an answer? Is this feature possible?
